### ISSUE TITLE
training is slow

### ISSUE THREAD

**Issue by xjsxujingsong** [YuliangXiu/MobilePose-pytorch#12]
Hi I am using the default mpii dataset and all the default parameters for training. With mobilenet/resnet, it takes around 8 seconds for each batch (including reading files and bp) with Titan V/1080Ti. Is there anything wrong?
For mobilenet, I already trained 300 epoch, with validation loss around 600. I tested the performance with images from mpii. Almost all the results are in the center of the image like standard position. The same thing for resnet model.
BTW, in the code during inference for resnet, the function "rescale" is slightly different from the one for training, resulting 216, 217 (h!=w) sized image.
The other thing is why use 5 channels in the first layer?

**Comment by YuliangXiu** [comment]
I can not remember the details about training time, the only thing I can remember is that I spent around 3~4 hours(1080Ti) training ResNet18 and get around 400 loss. MobileNet is around 3x slower, maybe this is because the unofficial implementation of MobileNet V2 is not efficient.
The current version of MobilePose is pretty naive, so the model can only deal with single full body person with the pre-located bounding box, you can test this model in single person image cropped from original MPII images.
In #4, I explain the reason why I use 5 channels.

**Comment by xjsxujingsong** [comment]
@YuliangXiu  THX. I will try resnet again to check the training speed and performance.
